src/index.html

```diff
@@ -6,7 +6,316 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body id="home">
+    <div class="container">
+      <header class="header">
+        <a href="#home" class="header__logo-link">
+          <img
+            class="header__logo-img"
+            src="images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#" class="nav__link is-active">Apple</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Smartphones</a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Gadgets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Tablets</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Photo</a>
+            </li>
+            <li class="nav__item">
+              <a href="#" class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <main class="wrapper-card">
+        <div class="card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          >
+          <div class="card__info">
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+            <p class="card__code">Product code: 195434</p>
+
+            <div class="card__rate">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="card__review">Reviews: 5</p>
+            </div>
+
+            <div class="card__price">
+              <div class="card__price-name">Price:</div>
+              <div class="card__price-amount">$2,199</div>
+            </div>
+
+            <div class="card__button">
+              <a
+                href="#"
+                class="card__button-link"
+                data-qa="card-hover">
+                  Buy
+              </a>
```

Comment:
Fix all cases if present
```suggestion
              <a
                href="#"
                class="card__button-link"
                data-qa="card-hover"
              >
                  Buy
              </a>
```